app/assets/stylesheets/sufia/_file-listing.scss

```diff
@@ -170,6 +178,10 @@ input#check_all {
   margin-bottom: 10px;
 }
 
+#appliedParams.constraints-container {
```

Comment:
Same comment about using ID selectors as above. Also, I am unsure where this is located or what it controls. Where is this used?